src/Media/MediaUrlRepository.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Media;
+
+final class MediaUrlRepository
+{
+    private array $mapping;
+
+    public function __construct(array $mapping)
+    {
+        $this->mapping = $mapping;
+    }
+
+    public function updateUrl(string $oldUrl): string
+    {
+        foreach ($this->mapping as $udb_variant => $settings) {
```

Comment:
The name of variables always follows the camelCasing convention and does not use underscores.

Reply:
If you use the name `mediaUrlMappings` as the private property then `mediaUrlMapping` is also a better and more meaningful name then `settings`